openshift/README.md

```diff
@@ -0,0 +1,40 @@
+## Registry CI/CD pipline setup.
+
+Regsitry app has a full CI/CD pipline using GitHub Action and Argo CD. and its been hosted on **Silver** Cluster.
+
+**CI**: [Github Action ](https://github.com/bcgov/platform-services-registry/actions) will trigger image build accordingly and commit to the [tenant-gitops-platform-registry](https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app) based on the target deployment environment(the job triggered).
+
+**CD**: [ArgoCD resource](https://argocd-shared.apps.silver.devops.gov.bc.ca/applications?proj=&sync=&health=&namespace=&cluster=&labels=&search=platform-registry) is will compare the current deployment configration and the record we have in[tenant-gitops-platform-registry](https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app) and do the deploy the application accordingly.
+
+### Environment has different usage
+
+- [Dev](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-dev): Testing env for feature branch
+- [Test](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-test): Testing env for main branch
+- [Prod](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-prod): Production environment for user.
+
+#### Usage of different env's pipeline
+
+The pricinple for all environment is the same but the way how to trigger/use the pielines for each env are slightly different.
+
+##### Dev
+
+We are using s2i strategy for regsitry web and api, because it is used for testing feature branch, we will then target the source to the feature branch that you want to test in the BuildConfig in tools namespaces under for both web([platsrv-registry-web-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-web-dev-build/yaml) ) and api([platsrv-registry-api-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-api-dev-build/yaml) ) under `spec.source.git.ref`(ie. update your branch name into that field). **DON'T FORGET TO SAVE YOUR CHANGE.**
```

Comment:
```suggestion
We are using s2i strategy for registry web and API. Because it is used for testing a feature branch, we will then target the source to the feature branch that you want to test in the BuildConfig in tools namespaces under for both web([platsrv-registry-web-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-web-dev-build/yaml) ) and api([platsrv-registry-api-dev-build](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/platform-registry-tools/buildconfigs/platsrv-registry-api-dev-build/yaml) ) under `spec.source.git.ref`(ie. update your branch name into that field). **DON'T FORGET TO SAVE YOUR CHANGE.**
```